### ISSUE TITLE
Boards & Cards: which are `/boards` and which are `/cards`

### ISSUE THREAD

**Issue by Pinjasaur** [wiggin77/mysilverbullets#1]
@wiggin77 It may be as simple as a typo/copypasta, but I'm not totally following which endpoints are nested under `/boards` and which are under `/cards`. For example:

get a board:

https://github.com/wiggin77/mysilverbullets/blob/5e693d8285e71d73532b152828b7da4f95387925/Boards%20API%20V3.md?plain=1#L24-L29

get card(s):

https://github.com/wiggin77/mysilverbullets/blob/5e693d8285e71d73532b152828b7da4f95387925/Boards%20API%20V3.md?plain=1#L90-L99

I may be misunderstanding, but they seem to be using the same HTTP method & path.

---

undelete a card:

https://github.com/wiggin77/mysilverbullets/blob/5e693d8285e71d73532b152828b7da4f95387925/Boards%20API%20V3.md?plain=1#L115-L120

As-is, would this be a no-op on a card that isn't already deleted? IMO, POST is more akin to the Create of CRUD.

Would it make sense to keep consistency between the equivalent endpoint for boards too? i.e. the `/undelete` suffix

https://github.com/wiggin77/mysilverbullets/blob/5e693d8285e71d73532b152828b7da4f95387925/Boards%20API%20V3.md?plain=1#L56-L61

**Comment by wiggin77** [comment]
Yup, good catch.  I missed the trailing verb in some of them.  Will fix.

**Comment by wiggin77** [comment]
Should be fixed via https://github.com/wiggin77/mysilverbullets/commit/dc6fb5cea9b4002c1f9af5006d2ea606de86d4bb   Thanks @Pinjasaur
